app/views/return-requirements/reason.njk

```diff
@@ -0,0 +1,26 @@
+{% extends 'layout.njk' %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% set title = "Select the reason for the return requirement" %}
+{% set rootLink = "/licences/" + licenceId %}
```

Comment:
you need the `/system/` at the beginning of the url

Reply:
thanks, missed that